README.md

```diff
@@ -105,12 +105,12 @@ and reboots the host if it's required (set `reboot_enabled` to `true`).
 
 #### Deploying updates
 
-`cd ansible && ansible-playbook -i inventory/production deploy-update.yml`
+`cd ansible && ansible-playbook -i inventory/production deploy-update.yml --tags update`
 
 Running the role with `reboot_enabled` will reboot a host that requires reboot
 after package upgrades:
 
-`cd ansible && ansible-playbook -i inventory/production deploy-update.yml --extra-vars=reboot_enabled=true`
+`cd ansible && ansible-playbook -i inventory/production deploy-update.yml --tags update --extra-vars=reboot_enabled=true`
```

Comment:
I would actually prefer to have the updates tag on the yaml file so running this role is like anyone else. It's separation of concerns. The calling of the role is regular, but it's the role itself that calls the update tags of the roles. Otherwise if one mistakenly forgets the `--tags update` it will fully run the roles. Kind of keeping it fail-safe.

Reply:
I would really like to not use the update tags explicitly.
@francisco-core Do you have any idea on how this could be implemented, perhaps I overlooked an option.

Reply:
I think it is something like

`cd ansible && ansible-playbook -i inventory/production deploy-mail.yml --tags update --extra-vars=reboot_enabled=true`

no need for an extra `deploy-update.yml` for each host.

Reply:
Resolved in: https://github.com/PrivacyLx/devops/pull/58/commits/81bfb8ac314fe40e693786f62f2a2f764a8908e3